msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/CacheRefreshReason.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+/**
+ * Specifies the reason for fetching the access token from the identity provider when using {@link AbstractClientApplicationBase#acquireTokenSilently(SilentParameters)}
+ */
+public enum CacheRefreshReason {
+    /**
+     * Token did not need to be refreshed, or was retrieved in a non-silent call
+     */
+    NOT_APPLICABLE(0),
+    /**
+     * Silent call was made with the force refresh option
+     */
+    FORCE_REFRESH(1),
```

Comment:
If telemetry logs 1, how do we distinguish between FORCE_REFRESH and CLAIMS

Reply:
FORCE_REFRESH and CLAIMS are mapped to the same telemetry value in .NET and (I think) other MSALs: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/Cache/CacheRefreshReason.cs

That is also the enum that [the telemetry schema docs](https://eng.ms/docs/microsoft-security/identity/entra-developer-application-platform/id4s-identity-for-services/authn-middleware-sdk-microsoft-identity-service-essentials/microsoft-identity-service-essentials/articles/using-mise/telemetry-schema) point to as an example of valid values. 

However, in a different comment Bogdan suggested splitting them in order to make the code/logs more clear.